extensions/spring-xd-extension-cassandra/src/main/java/org/springframework/xd/cassandra/mixins/CassandraConnectionMixin.java

```diff
@@ -44,7 +44,7 @@
 
 	private String initScript;
 
-	private String[] entityBasePackages = new String[0];
+	private String[] entityBasePackages;
```

Comment:
+1 for @ericbottard!

From other side we should revert this change for the reason I shown above.
Even if you tested it, we just don't have the test for this case. I had it before in the `modules` project, but @markpollack remove it here because it was `@Ignored` and didn't work from test environment by the ClassLoader issue.